templates/donations/donate.html

```diff
@@ -5,20 +5,54 @@
 <div id="content_full"> <!-- START of Content area -->
 
   <h1>Donations</h1>
-  <p>
-      Thanks for supporting Freesound!
-      <br>Please use the form below to make a donation. Note that donations are managed through the Phonos Foundation, an institution linked to the Music
-         Technology Group for the promotion of social and cultural initiatives. Click here to see the <a href="{% url 'donors' %}">full list of Freesound donors and and donations</a>.
-  </p>
-  <!-- donations -->
+    <p>
+        Thanks for supporting Freesound!
+    </p>
+    <p>
+        Please use the form below to make a donation. Note that donations are managed through the <b>Phonos Foundation</b>, an institution linked to the <b>Music
+        Technology Group</b> for the promotion of social and cultural initiatives. Click here to see the <a href="{% url 'donors' %}">full list of Freesound donors and and donations</a>.
+    </p>
+    <p>
+        Are you asking yourself "<b>why should I donate to Freesound?</b>"
+        <a href="javascript:void(0);" onclick="show_donate_reasons();">Here are some reasons</a>, let's see if we can
+        convince you :)
+    </p>
+
+    <div id="why_should_donate" style="display:none;">
```

Comment:
This section has good information, but it's hidden in some quite large paragraphs. What about being a bit more upfront with the reasons?

### Why support freesound? 
* **We are growing and need more resources.** [something about how we started and how we are maintained]
* **We don't like ads.** [because of the ethos of freesound we don't like ads and would prefer to ask for donations from the people who find value from the site]
* **We don't need many donations.** [just a small donation from everyone who uses freesound would help us grow and improve]